lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -196,6 +197,71 @@ def test_create_ccx(self):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(re.search('id="ccx-schedule"', response.content))
 
+    @SharedModuleStoreTestCase.modifies_courseware
+    @patch('ccx.views.render_to_response', intercept_renderer)
+    @patch('ccx.views.TODAY')
+    def test_get_ccx_schedule(self, today):
```

Comment:
@amir-qayyum-khan: I think I need to address first the issue you found with the "All subsections" selection.

Anyway @peter said that hiding an element at a higher level will hide all its children elements and the test I wrote actually tests this. 
(Or do you mean something else?)